docs/known_issues.md

```diff
@@ -60,3 +60,7 @@ velero backup create my-backup --include-namespaces my-namespace --snapshot-volu
 Users might require ways to provide registry credentials when deploying velero and vSphere plugin with images in private registries. Below are solutions in different cases.
 * In Vanilla cluster or Tanzu Kubernetes Grid Service cluster, users can [add the image pull secrets to the corresponding service accounts](https://kubernetes.io/docs/tasks/configure-pod-container/configure-service-account/#add-image-pull-secret-to-service-account).
 * In vSphere with Tanzu cluster, users can use the `--use-private-registry` option available in [velero vSphere operator CLI](velero-vsphere-operator-cli.md) to indicate pulling image from private registry. As a prerequisite, users are expected to provide the registry credential when enabling `Velero vSphere Operator` Supervisor Service.
+
+## Volume Expansion Not Supported when there are Snapshots
+
+If upload fails during a backup, the local snapshot will not be deleted. Velero vSphere Plugin will keep retrying the upload. If the user wants to expand volume, it will fail because volume expansion is not supported when there are snapshots on the volume.
```

Comment:
Do we want to recommend some solutions here?, like deleting the FCD snapshot from mob?. In the future release, they can even create the DeleteSnapshot CR.

Reply:
Discussed about this.  We'll add the work around when the DeleteSnapshot CR approach becomes available in the next release.

Reply:
BTW, we should also mention: before applying any workaround to delete snapshot, no matter whether it is FCD DeleteSnapshot API or DeleteSnapshot CRD, user should make sure cancel the backup if upload is still retrying.

Reply:
Good point.  We should mention that when we provide a workaround.